Don't access the net to load public links.

--- keystone-2011.3.1/doc/source/conf.py.orig	2011-11-30 21:40:44.822527914 +0100
+++ keystone-2011.3.1/doc/source/conf.py	2011-11-30 21:42:16.279821522 +0100
@@ -46,7 +46,6 @@
               'sphinx.ext.coverage',
               'sphinx.ext.viewcode',
               'sphinx.ext.ifconfig',
-              'sphinx.ext.intersphinx',
               'sphinx.ext.pngmath',
               'sphinx.ext.graphviz',
               'sphinx.ext.todo']
